stage/tests/lms/test_demo.py

```diff
@@ -0,0 +1,25 @@
+from bok_choy.web_app_test import WebAppTest
+from ...pages.lms.login_lms import LmsLogin
+from ...pages.lms.dashboard_lms import DashboardPageExtended
+
+
+class DemoTest(WebAppTest):
+    """
+    Sample Test for logging in and navigating to Courseware page
+    """
+
+    DEMO_COURSE_USER = os.environ.get('USER_LOGIN_EMAIL')
```

Comment:
Ah I see we're already repeating this pattern. We should keep it somewhere else. How about a base class that sets up vars? In spite of what I said earlier, since we're already repeating a pattern, we should fix it up in this PR, not later.